examples/hivemq-swarm/configmap_monitoring.yaml

```diff
@@ -0,0 +1,2372 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: swarm-hivemq-swarm-dashboard
+  namespace: monitoring # Namespace where the prometheus-community/kube-prometheus-stack is installed
+  labels:
+    app.kubernetes.io/instance: swarm # Helm release-name of HiveMQ Swarm chart
```

Comment:
```suggestion
    app.kubernetes.io/instance: swarm # CUSTOMIZE: Helm release-name of HiveMQ Swarm chart
```

Reply:
These extra labels are not needed. Will remove them. Only the `grafana_dashboard` is needed.